src/scss/includes/panels/poi_panel.scss

```diff
@@ -16,6 +21,31 @@ $BLOCK_INDENT: 60px;
   display: none;
 }
 
+.poi_panel__content__card {
+  display: none;
+}
+
+.poi_panel__content__card__content {
+  float: left;
+  width: calc(100vw - (2 * $CARD_PADDING + $CARD_ACTION_WIDTH + $CARD_ACTION_RIGHT_MARGIN));
```

Comment:
As you said, you may need interpolation syntax to make variables work here
See https://stackoverflow.com/a/20236515/5634396

Reply:
Thanks for the information : ]
falso it's fixed